app.go2tv.go2tv.yml

```diff
@@ -0,0 +1,90 @@
+app-id: app.go2tv.go2tv
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: go2tv
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+  - name: "x264"
+    config-opts:
+      - --disable-cli
+      - --enable-shared
+    sources:
+      - type: archive
+        url: http://download.videolan.org/x264/snapshots/x264-snapshot-20191217-2245-stable.tar.bz2
+        sha256: b2495c8f2930167d470994b1ce02b0f4bfb24b3317ba36ba7f112e9809264160
+
+  - name: fdk-aac
+    config-opts:
+      - --enable-shared
+    sources:
+      - type: archive
+        url: https://github.com/mstorsjo/fdk-aac/archive/v2.0.3.tar.gz
+        sha256: e25671cd96b10bad896aa42ab91a695a9e573395262baed4e4a2ff178d6a3a78
+
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -fiv
+
+  - name: ffmpeg
+    config-opts:
+      - --enable-gpl
+      - --enable-nonfree
+      - --enable-libx264
+      - --enable-libfdk-aac
+      - --enable-encoder=libx264
+      - --enable-encoder=libfdk_aac
+      - --enable-decoder=h264
+      - --enable-decoder=aac
+      - --enable-muxer=flv
+      - --enable-muxer=mov
+      - --enable-demuxer=mov
+      - --enable-demuxer=mp4
+      - --enable-protocol=pipe
+      - --enable-protocol=file
+      - --enable-filter=format
+      - --enable-ffprobe
+      - --disable-debug
+      - --disable-doc
+      - --disable-ffplay
+      - --enable-shared
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-7.1.tar.xz
```

Comment:
7.1 is in the ffmpeg-full extension, why not use that?

Reply:
ty, I wasn't aware